cmd/eval.go

```diff
@@ -176,19 +183,20 @@ Set the output format with the --format flag.
 	evalCommand.Flags().VarP(&params.profileCriteria, "profile-sort", "", "set sort order of expression profiler results")
 	evalCommand.Flags().VarP(&params.profileLimit, "profile-limit", "", "set number of profiling results to show")
 	evalCommand.Flags().VarP(&params.prettyLimit, "pretty-limit", "", "set limit after which pretty output gets truncated")
+	evalCommand.Flags().BoolVarP(&params.fail, "fail", "", false, "exits with non-zero exit code on undefined result and errors")
 	setIgnore(evalCommand.Flags(), &params.ignore)
 
 	RootCommand.AddCommand(evalCommand)
 }
 
-func eval(args []string, params evalCommandParams) (err error) {
+func eval(args []string, params evalCommandParams) (exitCode int, err error) {
```

Comment:
[nit] Named return arguments invite naked returns -- what would you think we'd loose when changing these to `(int, error)`?

Reply:
Fine with me, I just continued the style here

Reply:
Feel free to get rid of the named returns here.